GiniVision/Classes/CameraViewController.swift

```diff
@@ -210,7 +220,7 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
         super.viewWillTransition(to: size, with: coordinator)
         
         let orientation = UIDevice.current.orientation
-        (previewView.layer as? AVCaptureVideoPreviewLayer)?.connection?.videoOrientation = orientationsMapping[orientation]!
+        (previewView.layer as? AVCaptureVideoPreviewLayer)?.connection?.videoOrientation = deviceOrientationsMapping[orientation] ?? interfaceOrientationsMapping[UIApplication.shared.statusBarOrientation]!
```

Comment:
Here, too, we should only use the statusBarOrientation (and interfaceOrientationsMapping) if there is no case where it delivers erronous results.